ui/src/templates/components/createFromTemplateOverlay/TemplateBrowserList.tsx

```diff
@@ -23,16 +23,18 @@ class TemplateBrowser extends PureComponent<Props> {
         autoSize={false}
         noScrollX={true}
       >
-        {templates.map(t => (
-          <TemplateBrowserListItem
-            key={t.id}
-            template={t}
-            label={t.meta.name}
-            onClick={onSelectTemplate}
-            testID={`template--${t.meta.name}`}
-            selected={_.get(selectedTemplateSummary, 'id', '') === t.id}
-          />
-        ))}
+        {templates
+          .sort((a, b) => (b.meta.name < a.meta.name ? 1 : -1))
```

Comment:
probably want a `string.toLocaleLowerCase` when evaluating these string.  also, we have a component which does this for you.  it's called `SortingHat`.  if you don't want to use that, might I suggest using lodash's `sortBy` or `orderBy`.  It just reads nicer.